### ISSUE TITLE
BCSQ documentation

### ISSUE THREAD

**Issue by jb621-star** [samtools/bcftools#1671]
There is a lack of documentation for the definitions that that BSCQ uses for consequence calling. For example, the [Ensembl calculated variant consequences documentation](https://m.ensembl.org/info/genome/variation/prediction/predicted_data.html)  defines all the tags that it uses, but their criteria/definitions seem slightly different from the annotations BSCQ uses. Could you provide a similar list of definitions?

**Comment by pd3** [comment]
The definitions should be the same. If you found any differences, can you please report them here?

**Comment by jb621-star** [comment]
> The definitions should be the same. If you found any differences, can you please report them here?
> 
> EDIT: At the second thought you probably requesting a complete list of consequences supported by the program. There are some consequences that `bcftools csq` does not do because they cannot be inferred from the GFF (e.g. regulatory variants) and others are not reported at all, for example intergenic or feature elongation variants.

Yes, a complete list is what I'm looking for. I'm using a software ([NemaScan](https://www.elegansvariation.org/mapping/perform-mapping/)) that utilizes BSCQ to annotate variants, but it seemed like a few of the ENSEMBL consequences were not present in their results, so I just wanted to get an idea of how large this discrepancy is. Thank you!

**Comment by pd3** [comment]
The list of supported consequence types is now added to the manual page https://github.com/samtools/bcftools/commit/594e9b2a7f587741726b34db80d3c854f38bc66c
